ignite/chainconfig/parser.go

```diff
@@ -0,0 +1,239 @@
+package chainconfig
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/ignite-hq/cli/ignite/chainconfig/common"
+	"github.com/ignite-hq/cli/ignite/chainconfig/conversion"
+	v0 "github.com/ignite-hq/cli/ignite/chainconfig/v0"
+	v1 "github.com/ignite-hq/cli/ignite/chainconfig/v1"
+	"github.com/ignite-hq/cli/ignite/pkg/xfilepath"
+	"github.com/imdario/mergo"
+)
+
+var (
+	// ErrCouldntLocateConfig returned when config.yml cannot be found in the source code.
+	ErrCouldntLocateConfig = errors.New(
```

Comment:
Can we move this to a new file called chainconfig/config.go?

Reply:
done